### PR TITLE
Updated theming.md with proper namings

### LINKED ISSUE #16096: More clearly separate the two different types of themes in theming.md

### PR DESCRIPTION
**Pull request by Ashray123**
Closes #16096
 Signed-off-by: Ashray123 <jha.ashray12@gmail.com>

### CODE REVIEW

**Comment by Ashray123** [comment]
@aaronraimist please review and let me know if some other changes were expected :)

File: docs/theming.md [hunk 1]
```diff
@@ -24,8 +25,8 @@ To define a theme for Element:
 In future, the assets for a theme will probably be gathered together into a
 single directory tree.
 
-Custom Themes
-=============
+To define a JSON based theme for Element:
+=========================================
 
 Themes derived from the built in themes may also be defined in settings.
```
Comment (aaronraimist) [inline docs/theming.md:31]:
```suggestion
When [`feature_custom_themes`](https://github.com/vector-im/element-web/blob/master/docs/labs.md#custom-themes-feature_custom_themes) is enabled, JSON based themes derived from the built in themes may also be defined in settings.
```

File: docs/theming.md [hunk 2]
```diff
@@ -4,11 +4,12 @@ Theming Element
...
 Element app via CSS & custom imagery.
 
-They are *NOT* co be confused with 'skins', which describe apps which sit on top
+They are *NOT* to be confused with 'skins', which describe apps which sit on top
 of matrix-react-sdk - e.g. in theory Element itself is a react-sdk skin.
 As of Jan 2017, skins are not fully supported; Element is the only available skin.
 
-To define a theme for Element:
+To define a CSS based theme for Element:
+=========================================
```
Comment (aaronraimist) [inline docs/theming.md:12]: I think I would change the very first line to be "CSS Based Themes:" rather than changing this line.

**Review by aaronraimist — COMMENTED** [review]
Looks good but I just made a little tweak. This is how I would do it I think.

**Comment by aaronraimist** [comment]
Also somewhat unrelated but I noticed that the instructions there about how to do CSS themes seem to be out of date. Filed https://github.com/vector-im/element-web/issues/16804 for that.

**Pull request closed without merging**